cmd/api/src/api/v2/auth/auth.go

```diff
@@ -388,9 +388,13 @@ func (s ManagementResource) UpdateUser(response http.ResponseWriter, request *ht
 		api.WriteErrorResponse(request.Context(), api.BuildErrorResponse(http.StatusBadRequest, "a user can only have one role", request), response)
 	} else if roles, err := s.db.GetRoles(request.Context(), updateUserRequest.Roles); err != nil {
 		api.HandleDatabaseError(request, response, err)
-	} else if isNewEmail := s.db.IsNewEmail(request.Context(), updateUserRequest.EmailAddress); !isNewEmail {
-		api.WriteErrorResponse(request.Context(), api.BuildErrorResponse(http.StatusConflict, api.ErrorResponseUserDuplicateEmail, request), response)
 	} else {
+		// Prevent duplicate emails, this can be removed once the db constraint is added
+		if !strings.EqualFold(user.EmailAddress.String, updateUserRequest.EmailAddress) && !s.db.IsNewEmail(request.Context(), updateUserRequest.EmailAddress) {
+			api.WriteErrorResponse(request.Context(), api.BuildErrorResponse(http.StatusConflict, api.ErrorResponseUserDuplicateEmail, request), response)
+			return
+		}
+
```

Comment:
This could have been chained (which would have also removed the need for the return), but if it's going to be removed soon anyway, I don't know that it's worth changing now